src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java

```diff
@@ -168,6 +170,28 @@ public String toString() {
                 + ", answer=" + getSerializedFeedbackResponseDetail() + "]";
     }
 
+    @Override
+    public int hashCode() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(this.feedbackSessionName).append(this.courseId)
+                .append(this.feedbackQuestionId).append(this.giver).append(this.recipient);
+        return stringBuilder.toString().hashCode();
+    }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Don't forget to write test for the equals method.

Reply:
Added tests for the equals method